packages/core/nuxt-theme-module/theme/components/AppHeader.vue

```diff
@@ -50,6 +114,9 @@ export default {
     const { cart, loadCart } = useCart();
     const { loadWishlist } = useWishlist();
     const term = ref(getFacetsFromURL().term);
+    const currentCategory = ref('');
+    const isVisible = ref(true);
```

Comment:
is what visible?

if this is not a property that correspond to visibility fo `AppHeader` put a name of an element that is visible or not eg `isMegaMenuVisible`

Reply:
I think it was redundant property so I decided to get rid of it